.eslintrc.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
         },
     },
     env: {
+        es2020: true,
```

Comment:
My opinion: We should discuss the minimum required ES version alltogether (also with @PhilippeMarc-IGN and @LoicUTC) to check the impacts of bumping the minimum required ES version (browser support, does it have an impact for any of the users). This should be done in a public github discussion and conclusions of vocal discussions on this matter (if any) should also be written in this discussion. Regarding `BigInt`, if we find a quick and easy way to polyfill them, let's do it, otherwise consider not using them or wait on the decision of the ES version. WDYT?

Reply:
Now I use [jsbi](https://github.com/GoogleChromeLabs/jsbi) (thx to Desplandis) for BigInt polyfill with Babel and it works !